server/routes/index.ts

```diff
@@ -1,13 +1,23 @@
 import type { RequestHandler, Router } from 'express'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
+import CommunityApiService from '../services/communityApiService'
+import IntegrationSamplesRoutes from './integrationSamples'
 
-export default function routes(router: Router): Router {
+interface RouteProvider {
```

Comment:
trying to make a start on nailing down exactly how we want to split out our controllers.

we can make this more specific, e.g. we could map http verbs (an enum of get,put,delete etc) to request handlers or whatever we want. 

this is just to get us thinking about it